openex-api/src/main/java/io/openex/migration/V3_3__Delete_fk_users_injects.java

```diff
@@ -0,0 +1,26 @@
+package io.openex.migration;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+import java.sql.Connection;
+import java.sql.Statement;
+
+@Component
+public class V3_3__Delete_fk_users_injects extends BaseJavaMigration {
```

Comment:
Why the number V3_3 ?

Reply:
At first i thought that it is linked to the version we release but seems not. I will fix it

Reply:
Could be a good nomenclature !

Reply:
Yes but if one time we cannot release a migration on the version we thought it could lead to other problems

Reply:
https://www.red-gate.com/blog/database-devops/flyway-naming-patterns-matter

I not sure to understand which can of problem do you have in mind